src/test/kotlin/de/smartsquare/squit/mediatype/xml/XmlCanonicalizerSpek.kt

```diff
@@ -64,5 +64,37 @@ object XmlCanonicalizerSpek : Spek({
                 result shouldBeEqualTo structure
             }
         }
+
+        on("canonicalize an xml structure with resolving invalid namespaces") {
+            // language=xml
+            val structure = """
+                <ns3:test xmlns:ns3='w3.org/2001/XMLSchema-instance'>
+                    <hello b="b">Abc</hello>
```

Comment:
Let's add an "a" property here after the "b" to check that canonicalization works after resolving the namespace (see the "canonicalizing an xml structure -> should produce a valid result" spec).